lib/apiservers/portlayer/swagger.json

```diff
@@ -2408,6 +2686,138 @@
 					}
 				}
 			}
+		},
+		"LoggingJoinConfig": {
+			"type": "object",
+			"required": [
+				"handle"
+			],
+			"properties": {
+				"handle": {
+					"type": "object"
```

Comment:
@hmahmood we started with string because a serialized form can be represented as a string - after asking Ivan about how to return an untyped value he pointed us at this form.
If the Go API returns `interface{}` then it's simple to have the base layer returning rich objects, have another layer returning serialized forms, have another layer returning "pointers" as we are atm.
If we use string directly then we lose either the flexibility or the ability to layer shims of various types over the basic impl.